TESTING.md

```diff
@@ -37,6 +37,11 @@
     - [ ] [Colour scheme changes](#colour-scheme-changes)
 - Context
     - [ ] [PLG / SYMF / Local Keyword search](#local-keyword-search)
+- Model dropdown
+    - [ ] [Pro Account](#pro-account)
+    - [ ] [Enterprise account](#enterprise-account)
```

Comment:
can you move this entry to the end? it sounds more reasonable to group the items - dot com first, enterprise later